at-hoangtran/Week3/Day3/Exercise1-Day2.rb

```diff
@@ -0,0 +1,19 @@
+puts 'Input:'
+loop do
+  a = Float(gets.chomp)
+  break if a < 0 || a > 10
+  puts case a
+    when a >= 9
+      puts 'A'
+    when a >= 8
+      puts 'B'
+    when a >= 7
+      puts 'C'
+    when a >= 6
+      puts 'D'
+    else
+      puts 'F'
+  end
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/a5ed3ba5b6a204f5dbf8b2b54972d06d02f2dcdb/at-hoangtran/Week3/Day3/Exercise1-Day2.rb#L16"></span>Layout/EndAlignment: <code>end</code> at 16, 2 is not aligned with <code>case</code> at 5, 7.</td>
    </tr>
  </tbody>
</table>